framework/src/org/checkerframework/framework/type/AnnotatedTypeFactory.java

```diff
@@ -1081,12 +1069,22 @@ public AnnotatedTypeMirror fromMember(Tree tree) {
     }
 
     /**
-     * Determines the annotated type of an expression.
+     * Creates an AnnotatedTypeMirror for an ExpressionTree.
+     * The AnnotatedTypeMirror contains explicit annotations written with on the expression,
+     * annotations inherited from class declarations, and for some expressions, annotations
+     * from sub-expressions that could have been explicitly written, implicited, defaulted,
+     * refined, or otherwise computed.
+     *
+     * For example, the AnnotatedTypeMirror returned for an array access expression is the
+     * fully annotated type of the array component of the array being accessed.
      *
      * @param tree an expression
-     * @return the annotated type of the expression
+     * @return AnnotatedTypeMirror of the expressions either fully-annotated or partialy
+     * annotated depending on the kind of expression
```

Comment:
This helps me to know that this method has weird behavior that I should be paying close attention to, but doesn't tell me exactly what it does.